mender.go

```diff
@@ -64,8 +64,9 @@ const (
 )
 
 var (
-	defaultManifestFile = path.Join(getConfDirPath(), "build_mender")
-	defaultDataStore    = getStateDirPath()
+	defaultArtifactInfoFile = path.Join(getConfDirPath(), "artifact-info")
+	defaultDeviceTypeFile   = path.Join(getStateDirPath(), "device_type")
```

Comment:
Should we unify this somehow, so that we are having `artifact_info` and `device_type` or `artifact-info` and `device-type`?